src/data/symbol/symbol.tsv

```diff
@@ -359,6 +359,10 @@ POS	CHAR	Reading (space separated)	description	additional description	category	m
 アルファベット	э	きりる ろしあ えー	キリル文字	小文字
 アルファベット	ю	きりる ろしあ ゆー	キリル文字	小文字
 アルファベット	я	きりる ろしあ やー	キリル文字	小文字
+句読点	·	てん なかてん なかぐろ 、 ・ /	中点	欧文		OTHER	U+00B7 MIDDLE DOT
```

Comment:
Please let me know the reason why `/` is also a reading?

Reply:
It is just because KATAKANA MIDDLE DOT has the same reading. I wonder why it has the reading. I may as well remove it from the latin MIDDLE DOT and bullets.